VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs

```diff
@@ -124,16 +127,44 @@ protected override void OnUpdate()
 
 										SetComponent(coll.Entity, plungerMovementData);
 									}
+								break;
+								case ColliderType.Line:
+								case ColliderType.Line3D:
+								case ColliderType.Circle:
+								case ColliderType.LineZ:
+								case ColliderType.Plane:
+								case ColliderType.Point:
+								case ColliderType.Triangle:
+									// hit target
+									if (coll.Header.ItemType == ItemType.HitTarget) {
+										var normal = coll.Type == ColliderType.Triangle
+											? ((TriangleCollider*)collider)->Normal()
+											: collEvent.HitNormal;

```

Comment:
`normal` isn't used, right? Can I remove it? (I'm cleaning up a few other things, too.)

Reply:
yeah, screw it.